src/Authentication/GraphPhpLeagueAccessTokenProvider.php

```diff
@@ -24,14 +27,37 @@
  */
 class GraphPhpLeagueAccessTokenProvider extends PhpLeagueAccessTokenProvider
 {
+    public const NATIONAL_CLOUD_TO_AZURE_AD_ENDPOINT = [
+        NationalCloud::GLOBAL => 'https://login.microsoftonline.com',
+        NationalCloud::US_GOV => 'https://login.microsoftonline.us',
+        NationalCloud::CHINA => 'https://login.chinacloudapi.cn'
```

Comment:
I think we should also add the Germany one as the Azure folks have. 
https://github.com/Azure/azure-sdk-for-net/blob/1d3d91cf0dbe0c71cc1fa3753ca284674b19a513/sdk/identity/Azure.Identity/src/AzureAuthorityHosts.cs#L15

Reply:
Looks like it was deprecated - https://github.com/microsoftgraph/microsoft-graph-docs/pull/17947

Reply:
Happy to keep it as is then. 👍🏼